src/lib/components/Modal/README.mdx

```diff
@@ -363,58 +934,47 @@ component.
 
 ### Long Content and Autofocus
 
-👉 If you set the `scrollMode` to `modal`, you may want to turn `autoFocus` off
-to prevent the modal from scrolling to the end immediately after being opened.
+👉 If you wrap ModalContent with ScrollView, you may want to turn `autoFocus`
+off to prevent the modal from scrolling to the end immediately after being
+opened.
 
-## Blocking Modals
-
-There are situations when you need to interrupt user's flow and lock entire UI
-to prevent interaction. That's where blocking modals may come handy.
-
-<Playground>
-  {() => {
-    const [modalOpen, setModalOpen] = React.useState(false);
-    return (
-      <>
-        <Button
-          label="Launch blocking modal"
-          onClick={() => setModalOpen(true)}
-          priority="outline"
-        />
-        <div>
-          {modalOpen && (
-            <Modal title="Update is in progress" size="auto">
-              <p>
-                <span role="img" aria-label="Warning">⚠️</span>
-                Please wait a few seconds until the firmware update is finished.
-                Don&apos;t turn off the device!
-              </p>
-              <Button
-                label="Close the demo"
-                onClick={() => setModalOpen(false)}
-                priority="flat"
-              />
-            </Modal>
-          )}
-        </div>
-      </>
-    );
-  }}
-</Playground>
+<!-- markdownlint-disable MD024 -->
```

Comment:
We have duplicate headings because of Composition section. Any idea how we want to solve it? if we add API postfix, it should be added across whole documentation.

Reply:
I think it's alright the way it is!